first.css

```diff
@@ -1,3 +1,3 @@
-.header{
-    color: red;
-}
\ No newline at end of file
+.header {
+  color: red;
+}
```

Comment:
- [ ] **[REQUIRED]** Your commit messages still needs some improvement to make them more professional. Please read [here ](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/)about writing good commit messages.